app/assets/typescript/index.ts

```diff
@@ -124,6 +128,16 @@ function showAvailableMove(player: string, command: AvailableCommand) {
 
       break;
     }
+
+    case Command.Leech: {
+      addButton("Charge power: " + command.data, `${player} ${Command.Leech} ${command.data}`, {leech: command.data});
+      break;
+    }
+
+    case Command.DeclineLeech: {
+      addButton("Decline charge power", `${player} ${Command.DeclineLeech}`, {leech: command.data});
```

Comment:
There's no need for `{leech: command.data}`

Reply:
ok